content/03.methods.md

```diff
@@ -398,6 +398,27 @@ Craniopharyngiomas (CRANIO) were subtyped into adamantinomatous (`CRANIO, ADAM`)
 
 A molecular subtype of `EWS` was assigned to any tumor with a _EWSR1_ fusion or with a `pathology_diagnosis` of `Ewings Sarcoma`. 
 
+Any low-grade glioma (LGAT) samples that contained `low-grade glioma/astrocytoma`, `ganglioglioma` or `subependymal giant cell astrocytoma` but not `dysembryoplastic neuroepithelial tumor` in the pathology diagnosis were subtyped based on SNV, fusion and CNV status.
```

Comment:
I would expect the abbreviation here to be LGG based on the terminology used. What's the correct terminology here? cc: @jharenza

Reply:
Where might appropriate citations from https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/790 ([Ryall et al. [1]](https://actaneurocomms.biomedcentral.com/articles/10.1186/s40478-020-00902-z), [Ryall et al [2]](https://www.sciencedirect.com/science/article/pii/S1535610820301513)) go?

Reply:
These abbreviations are always hard, and I initially used LGAT/HGAT because astrocytomas and gliomas all arise from astrocytes. Though, most of the field uses pLGG and pHGG (pediatric LGG or HGG). So for this, I would suggest using 
`Any low-grade astrocytic tumor (LGAT) that contained...` 

I think we can defer to clinicians (I will ask) whether we should use pLGG or LGAT.

Reply:
```suggestion
Low-grade astrocytic tumors (LGAT) or glialneuronal tumors (GNT) were subtyped based on SNV, fusion and CNV status based on @doi:10.1016/j.ccell.2020.03.011, and as described below.
```

I took out some details